docs/eve-images.md

```diff
@@ -9,38 +9,176 @@ to use a different version, you may want to use a different disk image entirely.
 This is especially useful as part of the EVE development lifecycle.
 
 Retrieving disk images almost always is performed as part of `eden setup`.
-The actual launching of an image is part of `eden start`.
+The actual launching of an image is either part of `eden start` (when running
+local qemu) or performed externally (for external devices).
 
-## Where Eden normally gets EVE
+## Eden EVE Image Steps
 
-The normal EVE flow for `eden setup` is as follows.
+In the [Running Eden](../README.md#Running_Eden) guide, we described the
+image setup steps within a context as follows:
 
-1. If you do not have a context, get the latest tag for the image `lfedge/eve`
-from the docker hub, e.g. `0.0.51`, and save that tag in your context
-1. Pull the image `lfedge/eve:<tag>`
-1. Generate a config directory (inside `$PWD/dist/<name>-certs/`, where `<name>`
-the current context name) with:
-   * server certificate for adam
-   * `server` file pointing to the local running adam address
-   * device certificate
-1. Run the docker image with the config partition mounted to:
-   1. Generate a disk image - raw for RPi, qcow2 for everything else - overriding
-   the config partition with the mounted config directory
-   1. Extract the disk image from the docker image, e.g. `live.qcow2`
-1. Save the extracted disk image to a local cache, normally
-`$PWD/dist/<name>-images/eve/` (where `<name>` is the current context name).
+1. create a named context to store all of your configuration - `eden config add <name>`
+1. (optional) set options for the context - `eden config set <name> [options...]`
+1. run setup - `eden setup`, which extracts an eve-os qcow2 disk image from the docker image named in the context
```

Comment:
Seems, you also should add `--config <name>` here and below.

Reply:
Added

Reply:
Thank you!